### PR TITLE
Sms campaign  signup form

### LINKED ISSUE #2178: SMS Game: Step 2 styles
- [x] Alpha/Beta Signup Form
- [x] General template (help / sponsor footer)

### LINKED ISSUE #2408: [SMS game] - add CTIA messaging
Taking part in this experience means you agree to our Terms of Service & to receive our weekly update. Message & data rates may apply. Text STOP to opt-out, HELP for help.

Link Terms of Service to https://www.dosomething.org/about/terms-service.

Language should just appear exposed (i.e. not like Sources is treated) for now.

Should visually appear like it does when creating an account:

image

### PR DESCRIPTION
**Pull request by weerd**
Resolves #2408
Resolves #2178
Resolves #2408

@DFurnes

### CODE REVIEW

**Comment by DFurnes** [comment]
Looks beauteous. :nail_care:

**Comment by weerd** [comment]
haha, excellent emoticon.

File: lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--sms-game.tpl.php [hunk 1]
```diff
@@ -113,6 +113,10 @@
           <?php print render($signup_form); ?>
         <?php endif; ?>
       </div>
+
+      <div class="disclaimer legal">
+        <p>Taking part in this experience means you agree to our <a href="https://www.dosomething.org/about/terms-service">Terms of Service</a> &amp; to receive our weekly update. Message & data rates may apply. Text STOP to opt-out, HELP for help.</p>
```
Comment (aaronschachter) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--sms-game.tpl.php:118]: Not a blocker but lets get into the habit of using the `l` function for all links.

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--sms-game.tpl.php:118]: Oh, the `&` should be an `&amp;` as well.

Comment (weerd) [inline lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign--sms-game.tpl.php:118]: Crappola. Yeah, I forget about the "Drupal-way" somryimrd; I'll make a note and add it during a cleanup run. 

Also, yeah. I caught the one, forgot the other `&`! :cry:

**Pull request merged**